cmd/datamon/cmd/config.go

```diff
@@ -15,14 +15,13 @@ import (
 type CLIConfig struct {
 	// bug in viper? Need to keep names of fields the same as the serialized names..
 	Credential string `json:"credential" yaml:"credential"` // Credentials to use for GCS
-	Config     string `json:"config" yaml:"config"`         // Config for datamon
+	Config     string `json:"config" yaml:"config"`         // Config bucket for datamon
 	Context    string `json:"context" yaml:"context"`       // Context for datamon
 }
 
 func newConfig() (*CLIConfig, error) {
 	var config CLIConfig
-	err := viper.Unmarshal(&config)
-	if err != nil {
+	if err := viper.Unmarshal(&config); err != nil {
```

Comment:
more nitpick: whether to create non-`if ...`-scoped errors or `func`-scoped errors is another ongoing stylistic decision.  i still waffle on whether to use `if <statement>; <cond> { <block> }` or strictly stick with `if <cond> { <block> }` forms.  i usually err with the former as a function of how clever i'm feeling, and since "don't be clever" is the general advice in the golang ecosystem, it feels like eschewing the setup statement everywhere would be the more sensible rule ... i still feel clever sometimes, however.